network-area-diagram/src/main/resources/nominalStyle.css

```diff
@@ -9,10 +9,12 @@
 .nad-pst-arrow {stroke: #6a6a6a; stroke-width: 4; stroke-linecap: round; fill: none}
 .nad-state-out .nad-arrow-in {visibility: hidden}
 .nad-state-in .nad-arrow-out {visibility: hidden}
-.nad-active path {stroke: none; fill: #546e7a}
 .nad-active {visibility: visible}
```

Comment:
you can remove this now, together with `.nad-reactive {visibility: visible}` and `.nad-current {visibility: visible}`. It doesn't make sense any more. This was written only to "show" the possibility to enable/disable some infos.

Reply:
Ok!